src/anchor.rs

```diff
@@ -15,41 +15,41 @@ pub enum Anchor {
 
 impl Anchor {
     #[inline]
-    pub(crate) fn anim_side(&self) -> f32 {
```

Comment:
I would like to keep `&self` on all funcs here as they do not carry "consume" meaning.